content/en/data_streams/java-manual-instrumentation.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Setup Data Streams Monitoring for Java through Manual Instrumentation
+private: true
+further_reading:
+    - link: '/integrations/kafka/'
+      tag: 'Documentation'
+      text: 'Kafka Integration'
+    - link: '/tracing/service_catalog/'
+      tag: 'Documentation'
+      text: 'Service Catalog'
+---
+
+<div class="alert alert-info">Manual instrumentation is available for Java. For a list of technologies supported in Java today, see <a href="/data_streams/#setup">Data Streams Monitoring Setup</a>.<br /><br />If you're interested in manual instrumentation for additional languages, reach out to support@datadoghq.com.</div>
+
+Data Streams Monitoring propagates context through message headers. If you use a message queue technology that is not yet supported by DSM, a technology without headers (such as Kinesis), or lambdas, use manual instrumentation to set up Data Streams Monitoring.
+
+### Manual instrumentation installation
+
+1. Ensure you're using the [Datadog Agent v7.34.0 or later][1].
+
+2. Declare supported types, for example:
```

Comment:
@janewang-datadog Where should they do this?

Reply:
on services sending messages or consuming messages